inscriptions/ISic000381.xml

```diff
@@ -190,40 +195,35 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141193" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141193" xml:lang="la" resp="#JH">
                 <ab>
                     <lb n="1"/>
-                    <expan>
-                        <abbr>D</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>acrum</ex>
-                    </expan>
-                    .<lb n="2"/>Valeria Thermitana vixit<lb n="3"/>annos III
-                    <expan>
-                        <abbr>mens</abbr>
-                        <ex>es</ex>
-                    </expan>
-                    IIII dies VI<lb n="4"/>Valerius Eutyches et Lysidia<lb n="5"/>Thaumasie parentes
-                    <expan>
-                        <abbr>fecerun</abbr>
-                        <ex>t</ex>
-                    </expan>
-                    <lb n="6"/>filiae piissimae<lb n="7"/>et sibi et suis.
+                        <expan><abbr>D</abbr><ex>is</ex></expan> · 
+                        <expan><abbr>M</abbr> <ex>anibus</ex> </expan> ·
+                        <expan><abbr>s</abbr><ex>acrum</ex></expan> ·
+                    <lb n="2"/><name>Valeria</name> <name>Thermitana</name> · vixit
+                    <lb n="3"/>annos <num value="3">III</num>
+                            <expan><abbr>mens</abbr><ex>es</ex></expan> <num value="4">IIII</num>
+                            dies <num value="6">VI</num>
+                    <lb n="4"/><name>Valerius</name> · <name>Eutyches</name> et <name>Lysidia</name>
+                    <lb n="5"/>Thaumasie parentes <expan><abbr>fecerun</abbr><ex>t</ex></expan>
+                    <lb n="6"/>filiae · piissimae
+                    <lb n="7"/>et sibi et suis
                 </ab>
                 <!--This EDR text 141193 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text of CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" resp="#JH">
+                <p>To the gods of the underworld,
+                Valeria Thermitana lived for
+                three years, four months, and six days.
+                Valerius Eutyches and Lysidia Thaumasie,
+                her parents, erected this (plaque) for
+                their most dutiful daughter,
+                for herself and her family.
```

Comment:
ah,t he delights of reflexive pronouns. This should be: 'made this (tomb?) for their most dutiful daughter, and for themselves and their family.'